url_spec.js fails if travis runs in the middle of the night

https://github.com/TryGhost/Ghost/blob/master/core/test/unit/utils/url_spec.js#L506

```
  1) Url urlPathForPost post is not published yet:
      AssertionError: expected '/2017/02/17/short-and-sweet/' to be '/2017/02/16/short-and-sweet/'
      + expected - actual
      -/2017/02/17/short-and-sweet/
      +/2017/02/16/short-and-sweet/
      
    at Assertion.fail (node_modules/should/cjs/should.js:258:17)
    at Assertion.Object.defineProperty.value (node_modules/should/cjs/should.js:335:19)
    at Context.<anonymous> (core/test/unit/utils/url_spec.js:517:55)
```

Affects all test runs, on mysql and sqlite.
The timezone is not set, this might be the reason?

Ghost: master (lts not tested)

--- a/core/test/unit/utils/url_spec.js
+++ b/core/test/unit/utils/url_spec.js
@@ -2,7 +2,7 @@
 var should = require('should'),
     sinon = require('sinon'),
     _ = require('lodash'),
-    moment = require('moment'),
+    moment = require('moment-timezone'),
     utils = require('../../../server/utils'),
     settingsCache = require('../../../server/settings/cache'),
     configUtils = require('../../utils/configUtils'),
@@ -505,10 +505,11 @@ describe('Url', function () {
         });
 
         it('post is not published yet', function () {
+            localSettingsCache.activeTimezone = 'Europe/London';
             localSettingsCache.permalinks = '/:year/:month/:day/:slug/';
 
             var testData = _.merge(testUtils.DataGenerator.Content.posts[2], {id: 3, published_at: null}),
-                nowMoment = moment(),
+                nowMoment = moment().tz('Europe/London'),
                 postLink = '/YYYY/MM/DD/short-and-sweet/';
 
             postLink = postLink.replace('YYYY', nowMoment.format('YYYY'));

